### DISCUSSION TITLE
Thumbnail for a particular face in people section shows error loading image

### DISCUSSION THREAD

**Discussion by vs30217** [immich-app/immich#14369] [General]
Under people section for 1 particular face it is not showing image but showing error loading image. However once it's clicked on that image it correctly fetches all the relevent image of the intended person. how it can be corrected so that on people section correct face can be displayed rather than error loading image.

immich server- 1.121.0
web UI

**Comment by Arnie97** [comment]
Would you try "Select featured photo", and see if it fix the problem for you?
